### PR TITLE
Fixed DE translation für 'Insertion case'

### PR DESCRIPTION
**Pull request by fjf2002**

### CODE REVIEW

**Review by E1izabeth — APPROVED** [review]

**Review by LonwoLonwo — APPROVED** [review]

File: plugins/org.jkiss.dbeaver.ui.editors.sql/src/org/jkiss/dbeaver/ui/editors/sql/internal/SQLEditorMessages_de.properties [hunk 1]
```diff
@@ -52,7 +52,7 @@ pref_page_sql_completion_label_enable_auto_activation_tip = Automatische Aktivie
 pref_page_sql_completion_label_folding_enabled = Faltung aktiviert
 pref_page_sql_completion_label_folding_enabled_tip = Faltung in SQL-Skripten verwenden
 pref_page_sql_completion_label_hide_duplicate_names = Verstecke doppelte Namen von nicht aktiven Schematas
-pref_page_sql_completion_label_insert_case = Einlegekategorie
+pref_page_sql_completion_label_insert_case = Groß-/Kleinschreibung beim Einfügen
```
Comment (E1izabeth) [inline plugins/org.jkiss.dbeaver.ui.editors.sql/src/org/jkiss/dbeaver/ui/editors/sql/internal/SQLEditorMessages_de.properties]:
```suggestion
pref_page_sql_completion_label_insert_case = Gro\u00df-/Kleinschreibung beim Einf\u00fcgen
```

**Comment by MashaKorax** [comment]
In the app interface, it looks like  this. Would you kindly make a fix?
dbeaver_B5cryjfogM

**Review by E1izabeth — CHANGES_REQUESTED** [review]
I suppose it should be like this.
Native characters doesn't fit into ascii encoding.

**Review by E1izabeth — APPROVED** [review]

**Comment by MashaKorax** [comment]
verified

**Pull request merged**